slo-monitor/src/monitors/pod_monitor.go

```diff
@@ -51,6 +52,14 @@ var (
 			Buckets: prometheus.LinearBuckets(0.5, 0.25, 50),
 		},
 	)
+
+	PodFullStartupLatency = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Name:    PodFullStartupLatencyKey,
+			Help:    "Pod e2e startup latencies in milliseconds, without image pull times",
+			Buckets: prometheus.LinearBuckets(0.5, 0.5, 100),
```

Comment:
Do we really need those many buckets with that width (compared to the other metric)?
I *might* be wrong but IIRC from a past discussion, histogram with more buckets costs server-side computations more for calculating %iles.

Reply:
It's additional 800 bytes. I really wouldn't worry about that.

Reply:
Well, if you say so :)

Reply:
There's probably some overhead, but we can live with few orders of magnitude of overhead in this case;). After all it's just 100 64-bit floats + mutex (or several).

Reply:
Yup, memory is one thing. But there's also 2x more computation involved in computing %ile with 100 buckets as compared to 50 buckets (in the worst case) - so it can be slower. Though I'm not sure how much it'll affect in practice - you probably can neglect it.

Reply:
It would be true it it were Summary. But that's Histogram, so no computation is happening.

Reply:
On client-side yes, but on server-side there is computation happening for percentiles (and that's a bit heavy). Ref -  https://prometheus.io/docs/practices/histograms/#quantiles

Reply:
I know nothing about that.